lib/ui/profile/Profile.dart

```diff
@@ -348,4 +438,196 @@ class Profile extends StatelessWidget {
       ),
     );
   }
+
+  List<ProfileListItemViewModel> _profileItems(
+    ProfileViewModel viewModel,
+    BuildContext context,
+  ) {
+    List<ProfileListItemViewModel> list = [];
+
+    list.add(ProfileListItemViewModel(
+      title: _LocalisedStrings.switchLanguage(),
+      icon: _Icons.language,
+      onTap: () => _switchLanguage(context, viewModel),
+      isDestructive: false,
+    ));
+
+    list.add(ProfileListItemViewModel(
+      title: _LocalisedStrings.yourFarmDetails(),
+      icon: _Icons.newProfile,
+      onTap: () => _openFarmDetails(viewModel, context),
+      isDestructive: false,
+    ));
+
+    list.add(ProfileListItemViewModel(
+      title: _LocalisedStrings.inviteFriends(),
+      icon: _Icons.inviteFriends,
+      onTap: () => _inviteFriends(),
+      isDestructive: false,
+    ));
+
+    list.add(ProfileListItemViewModel(
+      title: _LocalisedStrings.createNewProfile(),
+      icon: _Icons.soil,
+      onTap: () => _createNewProfile(viewModel, context),
+      isDestructive: false,
+    ));
+
+    list.add(ProfileListItemViewModel(
+      title: _LocalisedStrings.privacyPolicy(),
+      icon: null,
```

Comment:
What if we define this icon as null by default and then just not inform it? This will let us create view model without specify icon as null

Reply:
You're right

Reply:
done